.coveragerc

```diff
@@ -182,6 +182,8 @@ omit =
     homeassistant/components/devolo_home_control/sensor.py
     homeassistant/components/devolo_home_control/subscriber.py
     homeassistant/components/devolo_home_control/switch.py
+    homeassistant/components/devolo_home_network/device.py
```

Comment:
I don't know if the powers that be will accept excluding these two from coverage with a gold integration scale given "Tests for fetching data from the integration and controlling it" is one of that level's requirements.

Reply:
Agree, but I put all the data fetching into entity_classes.py or did I get the requirement wrong?

Reply:
I haven't seen other integrations do it that way, but *shrug*. The ones I've seen all use the real entities and a mock for the backing Python module. You'll have to get clarification from someone who is a Member.

Reply:
That's still the way my approach works, I just concentrated the logic in a different file. This does not make big a difference now, but will hopefully lead to less code repetition with upcoming platforms and features.

Reply:
Please follow our standard test pattern. Each platform should be tested in a file named `test_PLATFORM.py`.

Reply:
For gold or platinum level quality scale no module should be excluded from coverage.

Reply:
Since then, I feared, that I might got the instructions wrong. @MartinHjelmare , could you maybe look at #49843? If I've done it right there, I know what to do here.

Reply:
Hi @MartinHjelmare , I reworked the tests and they should follow the standard pattern now.